src/pages/overview/document-generation-api/inlineimages.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Document Services APIs | Document Generation API | Inline Images
+---
+# Inline Images
+
+Add inline images in the document through json data using the <img\> tag.
+
+## How It Works
+
+You can provide the input image url by making use of html img tag in json. This is one such sample json.
+
+```json
+{
+  "ContentBlock": "This is my chart:<br><img src=\"https://canvasjs.com/wp-content/uploads/images/gallery/php-charts/overview/php-charts-graphs-index-data-label.png\">"
+}
+```
```

Comment:
Explain here that,
In the above example, the tag `ContentBlock` contains an `<img/>` html tag with **src** value "https://canvasjs.com/wp-content/uploads/images/gallery/php-charts/overview/php-charts-graphs-index-data-label.png". This image will be downloaded and placed in the final document like shown below.